Tools/autotest/default_params/rover.parm

```diff
@@ -21,7 +21,7 @@ MODE5        2
 MODE6        0
 STEER2SRV_P     1.8
 SIM_GPS_DELAY 1
-NAVL1_PERIOD 6
+NAVL1_PERIOD 8
```

Comment:
that is the question I ask to @gmorph . in parameter the value is set to 6 but by default the value is set at 8 ... So what it the better ? I don't feel a difference on SITL and gazebo ... so I don't know